stdlib/typing.pyi

```diff
@@ -463,7 +463,11 @@ class Generator(Iterator[_YieldT_co], Generic[_YieldT_co, _SendT_contra, _Return
     @overload
     @abstractmethod
     def throw(self, typ: BaseException, val: None = None, tb: TracebackType | None = None, /) -> _YieldT_co: ...
-    def close(self) -> None: ...
+    if sys.version_info >= (3, 13):
+        def close(self) -> _ReturnT_co: ...
```

Comment:
Looks like it still returns None in the case where GeneratorExit is not caught. So I think we should include that in the return type:
```suggestion
        def close(self) -> _ReturnT_co | None: ...
```

Example:

```
>>> def f():
...     yield
...     return 3
...     
>>> g = f()
>>> g.send(None)
>>> g.close()
>>> def f2():
...     try:
...         yield
...     except GeneratorExit:
...         return 3
...         
>>> g2 = f2()
>>> g2.send(None)
>>> g2.close()
3
```